<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiMainWindow</class>
 <widget class="QMainWindow" name="uiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QDataVis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Function_tab">
       <attribute name="title">
        <string>Functions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QTextEditCustom" name="QTextEdit_functionInput">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Add function here</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Global Point density</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_setGlobalPointDensity">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="QListWidget_functionList"/>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="QPushButton_addFunction">
          <property name="text">
           <string>Add function</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="QPushButton_deleteFunction">
          <property name="text">
           <string>Delete function</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Points_tab">
       <attribute name="title">
        <string>Points</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0" rowspan="2">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QPushButton" name="QPushButton_AddPointGraph">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="QPushButton_RemovePointGraph">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget_PointGraphList"/>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="regression_box">
          <item>
           <widget class="QSpinBox" name="spinBox_regressionDegree">
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_regression">
            <property name="text">
             <string>Regression</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_settingsOpenGL">
          <property name="text">
           <string>Use OpenGL (requires restart)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_enableCursor">
          <property name="text">
           <string>Enable cursor</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_settingsDarkMode">
          <property name="text">
           <string>Dark Mode</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox_stickAxisToZeroLines">
          <property name="text">
           <string>Stick axis to zero line</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_centerPlot">
      <property name="text">
       <string>Center</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QCustomPlotCustom" name="customPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="actionImport_data"/>
    <addaction name="actionExport_data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionProperties"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGraph"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Show Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionImport_data">
   <property name="text">
    <string>Import data</string>
   </property>
  </action>
  <action name="actionExport_data">
   <property name="text">
    <string>Export data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlotCustom</class>
   <extends>QWidget</extends>
   <header>QCustomPlotCustom.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTextEditCustom</class>
   <extends>QTextEdit</extends>
   <header>QTextEditCustom.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
